enterprise/install-scripts/amazon/openshift-amz.sh

```diff
@@ -990,7 +990,7 @@ set_conf()
   local longcomment=
   (( $# > 0 )) && printf -v longcomment '# %s\\\n' "$@"
 
-  sed -i -e "
+  sed -i -e --follow-symlinks "
```

Comment:
This isn't going to work. The -e flag must be directly followed by the expression. The change probably should be:
```
sed -i --follow-symlinks -e
```